tests/e2e/scenarios/test_reborn_webui_v2_legacy_responses_api.py

```
@@ -0,0 +1,198 @@
+@pytest.fixture(scope="session")
+def ironclaw_reborn_openai_compat_binary():
+    """Build Reborn with the OpenAI-compatible Responses route mounted."""
+    subprocess.run(
+        [
+            "cargo",
+            "build",
+            "-p",
+            "ironclaw_reborn_cli",
+            "--features",
+            "openai-compat-beta",
+        ],
```

Ruff (0.15.18)

[error] Starting a process with a partial executable path

(S607)